### DISCUSSION TITLE
How to keep using volume control buttons

### DISCUSSION THREAD

**Discussion by weichselbaum** [ExistentialAudio/BlackHole#607] [Q&A]
Hello!

I am aware this might be a MacOS issue, but is there a way to keep using the volume control buttons to regulate volume, after setting a Multi-Output-Device as primary audio device?

Kind regards,
Lukas

**Comment by devinroth** [comment]
That’s on macOS.

Devin Roth

**Comment by weichselbaum** [comment]
Thanks Devin.

For anyone having that problem, here's a workaround:

Go to sound preferences and at the bottom tick "show volume in menu bar".
I just use the normal Output "Headphones" on my MacBook, and before recording I switch to the Multi-Output Audio Device.
That makes it very convenient and allows me to keep using my volume control hotkeys.

**Comment by BigBillete** [comment]
Which program are you using to record?
